### PR TITLE
add `--no-daemon` and `--spin-time` explanation in readme.

### PR DESCRIPTION
**Pull request by fujitatomoya**
address part of https://github.com/ros2/sros2/issues/306

### CODE REVIEW

File: SROS2_Linux.md [hunk 1]
```diff
@@ -120,24 +120,27 @@ To be able to use the ros2 CLI tools to interact with your secured system, you n
...
 
-Then use the CLI as usual:
+Then use the CLI with `--no-daemon` and `--spin-time`:
+
+> [!NOTE]
+> Avoid using `ros2 daemon` since it may not have security enclaves, and enough time duration should be given for the discovery in secured network.
 
 ```bash
-ros2 node list
+ros2 node list --no-daemon --spin-time 3
```
Comment (fujitatomoya) [inline SROS2_Linux.md]: 3 seconds come from my amd64 environment, it should give it enough time window for the security handshake and discovery. (actually, fast-dds security takes much longer time compared to cyclonedds, that is why we did not see this issue with cyclonedds. but possibly this is common issue.)

**Review by fujitatomoya — COMMENTED** [review]
@mikaelarguedas could you review this?

**Comment by fujitatomoya** [comment]
i think we want to backport this to jazzy at least.

File: SROS2_Linux.md (hunk 1 above)
Comment (mikaelarguedas) [inline SROS2_Linux.md]:
```suggestion
ros2 node list --no-daemon --spin-time 4
```

Unclear how the value was picked at the time, but in the integration tests the value used is 4 so maybe we should use the samme here ?
https://github.com/ros2/sros2/blob/07608ab784bac6b295e7c031692f9bc6598adf76/sros2/test/sros2/commands/security/verbs/utilities/sros2_cli_test_case.py#L32
https://github.com/ros2/sros2/blob/07608ab784bac6b295e7c031692f9bc6598adf76/sros2/test/sros2/commands/security/verbs/utilities/sros2_cli_test_case.py#L72-L74

Comment (fujitatomoya) [reply SROS2_Linux.md]: sounds good.

**Review by mikaelarguedas — APPROVED** [review]
:+1: lgtm and agree to backport to jazzy as this bit of the tutorial alreay made it to the Jazzy branch: https://github.com/ros2/sros2/pull/296

**Comment by mikaelarguedas** [comment]
@Mergifyio backport jazzy

**Pull request merged**